<!DOCTYPE HTML>
<!--
	Template: Tessellate by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>GEOL2301: Frontiers in palaeontology</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />

    <link rel="apple-touch-icon" sizes="180x180" href="icons/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="icons/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="icons/favicon-16x16.png">
    <link rel="manifest" href="icons/site.webmanifest">
    <link rel="mask-icon" href="icons/safari-pinned-tab.svg" color="#5bbad5">
    <link rel="shortcut icon" href="icons/favicon.ico">
    <meta name="msapplication-TileColor" content="#da532c">
    <meta name="msapplication-config" content="icons/browserconfig.xml">
    <meta name="theme-color" content="#ffffff">
	</head>
	<body class="is-preload">

		<!-- Header -->
    <section id="header" class="dark">
      <header>
        <h1>GEOL2301: Frontiers in palaeontology</h1>
        <p>Session 4: Molecular clocks</p>
      </header>
      <footer>
        <a href="#first" class="button scrolly">Begin</a>
      </footer>
    </section>


    <!-- First -->
    <section id="first" class="main">
      <header>
        <div class="container">
          <h2>Molecular clocks</h2>
          <p>
            This week we&rsquo;ll explore the journey from DNA sequences to
            dated phylogenies by performing our own molecular clock analyses.
          </p>
          <p>
            Before you start, download and unpack
            <a href="https://www.beast2.org/" target="_blank" title="Download BEAST">BEAST2</a>,
            <a href="https://github.com/beast-dev/tracer/releases/tag/v1.7.2" title="Download Tracer" target="_blank">Tracer</a>,
            and (Windows users only)
            <a href="https://www.java.com/en/" target="_blank" title="Download Java Runtime Environment">Java</a>.
            <br />
            (For Tracer, Look under “Assets” for the release appropriate to your computer:
            Windows = <code>Tracer.zip</code>; Mac = <code>.dmg</code>)
          </p>
        </div>
      </header>
      <div class="content dark style1 featured">
        <div class="container">
          <div class="row">
            <div class="col-4 col-12-narrow">
              <span class="feature-icon"><span class="icon solid fa-dice-one"></span></span>
              <p>Understand what controls the accumulation of mutations in species&rsquo; DNA</p>
            </div>
            <div class="col-4 col-12-narrow">
              <span class="feature-icon"><span class="icon solid fa-dice-two"></span></span>
              <p>Perform a molecular clock analysis</p>
            </div>
            <div class="col-4 col-12-narrow">
              <span class="feature-icon"><span class="icon solid fa-dice-three"></span></span>
              <p>Evaluate the assumptions and limitations of molecular clock results</p>
            </div>
            <div class="col-12">
              <footer>
                <a href="#second" class="button scrolly">Let&rsquo;s get started!</a>
              </footer>
            </div>
          </div>
        </div>
      </div>
    </section>

  <!-- Second -->
    <section id="second" class="main">
      <header>
        <div class="container">
          <h2>Molecular sequence data</h2>
          <p>
            From DNA sequences to dated phylogenies
          </p>
        </div>
      </header>
      <div class="content dark style1">
        <div class="container">
          <div class="row">
            <div class="col-8 col-12-narrow">
              <h3>How DNA makes proteins</h3>
              <p>
                DNA, bundled in the chromosomes within the nucleus of each cell,
                comprises a backbone of the sugar deoxyribose that holds
                in place a series of nucleotide <strong>bases</strong>,
                typically referred to by their initial letters:
                adenine (A), cytosine (C), guanine (G) and thymine (T).
              </p>

              <p>
                A sequence of three bases forms a <strong>codon</strong>, which
                encodes a specific amino acid.
                Notice that the base in the third position of a codon is often
                redundant – this base can often be changed without affecting
                the translation.
              </p>
              <img src="images/clock/Codon-Chart-2.svg" />

              <p>
                Some sections of DNA encode <strong>genes</strong>.
                In animals, genes start with an <code>ATG</code> codon, and end
                with one of three "STOP" codons.
                An enzyme, RNA polymerase, binds to the start codon (perhaps
                with the help of transcription factors that bind to the DNA
                sequence before the start of the gene) and makes
                messenger RNA copies of the nucleotide sequence.
              </p>
              <p>
                <strong>Ribosomes</strong> then bind to the start of the mRNA
                sequences and work their way along.
                At each codon position, they bind the corresponding amino acid
                to a growing chain.  When they reach the STOP codon at the end
                of the mRNA sequence, the completed amino acid sequence folds
                to form a protein, often with a complex three-dimensional
                structure.
              </p>

              <p>
                Mutations cause changes to individual nucleotide bases within
                a DNA sequence – or may cause bases to be added or lost.
                What impact might mutations have on the resulting protein?
              </p>

              <ul>
                <li class="question" onclick="Reveal('mutate-base');">Changing an individual base</li>
                <li class="bulb">Think about the specific codons <code>TTG</code> and <code>TGT</code></li>
                <li class="hidden written answer" id="mutate-base">
                  <p>If the base is in the third codon position, often there will be no effect.
                  Some first-position changes have no effect, too.
                  </p><p>
                  Changes in the second position usually change the amino acid.
                  Amino acids with similar chemical properties often share similar codes –
                  but a change in an amino acid in the active site of an enzyme could
                  render the protien non-functional.
                  </p><p>
                  Cystine (Cys) contains sulfur and forms strong S-S bonds that often
                  control how a protein folds; removing these structural elements may be
                  more catastrophic than swapping two amino acids with similar chemistry.
                  </p>
                </li>
                <li class="question" onclick="Reveal('add-base');">Adding a new base</li>
                <li class="hidden written answer" id="add-base">
                  This will shift the reading frame; every codon after the insertion will encode a different amino acid.
                  This is likely disastrous for the protein, as the sequence of amino acids will be essentially random.
                </li>
                <li class="question" onclick="Reveal('lose-bases');">Losing a series of bases</li>
                <li class="hidden written answer" id="lose-bases">If the number of bases lost is a multiple of three, the odds of a functional output are higher, as the reading frame is not shifted.</li>
              </ul>

              <p>
                Deleterious mutations – those that decrease the fitness of an
                individual – are unlikely to persist in the gene pool.
                Hence the rate at which (germ line) mutations occur is higher
                than the rate at which <em>persistent</em> mutations are added
                to the gene pool of a population or species.
              </p>

              <p>
                If this latter rate is approximately uniform over time, we might
                in principle use it to deduce how recently two lineages diverged.
              </p>

              <ul>
                <li class="question" onclick="Reveal('mol-calc');">
                  Assume that one mutation modifies a particular gene's sequence
                  every million years.  Two species have sequences that
                  differ in 50 places.  Based on a quick calculation, estimate
                  how many years may have elapsed since their last common ancestor.
                </li>
                <li class="bulb">It may help to draw a simple phylogenetic tree</li>
                <li class="hidden written answer" id="mol-calc">
                  <p>
                  A good first guess might be 25&nbsp;million years.
                  The two species each followed separate evolutionary pathways
                  for 25&nbsp;Ma; the <strong>ancestral</strong> sequence would
                  differ from each living descendant in 25 places.
                  </p>
                </li>
                <li class="question" onclick="Reveal('mol-more');">
                  Why might the true divergence date be higher than a simple
                  calculation would suggest?
                </li>
                <li class="bulb">
                  Consider a case where the common ancestor had an
                  <code>A</code> at a specific position.
                  Think of scenarios where mutations may have occurred, but
                  not be observable based on the sequences available.
                </li>
                <li class="hidden written answer" id="mol-more">
                  <p>
                    If our ancestor has <code>A</code>:
                    <ul>
                      <li>
                        One descendant may have mutated to <code>C</code>,
                        then mutated again to <code>G</code>.
                        We'd count one mutation where two had occurred.
                      </li>
                      <li>
                        One descendant may have mutated to <code>C</code>,
                        then mutated again to the original <code>A</code>.
                        We'd count no mutations, where two had occurred.
                      </li>
                      <li>
                        One descendant might have mutated to <code>C</code>,
                        the other to <code>G</code>.
                        We'd count one mutation where two had occurred.
                      </li>
                      <li>
                        Both descendants might have mutated to <code>C</code>.
                        We'd count no mutations, where two had occurred.
                      </li>
                    </ul>
                  </p>
                </li>

                <li class="question" onclick="Reveal('mol-const');">
                  How confident do you feel that we can assume a single, constant
                  mutation rate across all taxa?
                </li>
                <li class="hidden written answer" id="mol-const">
                  Many factors might work against this – population size;
                  generation time (i.e. opportunities for mutations to occur);
                  plasticity of organisms; whether taxa are generalists or specialists
                  (likelihood that deleterious mutations might be overcome);
                  mass extinctions or evolutionary radiations
                  (new opportunities may change selection constraints).
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </section>

    <!-- Third -->
    <section id="third" class="main">
      <header>
        <div class="container">
          <h2>A first molecular clock analysis</h2>
          <p>
            Conducting a simple analysis to explore the operation of the molecular clock
          </p>
        </div>
      </header>
      <div class="content dark style1">
        <div class="container">
          <div class="row">
            <div class="col-8 col-12-narrow">
              <h3>Objectives</h3>
              <p>
                We will conduct a simple molecular clock to estimate the timing
                of mammal origins, using sequences of the gene catalase –
                a core cellular enzyme responsible for breaking down the
                "free radical" (oxidative stressor) hydrogen peroxide
                (H<sub>2</sub>O<sub>2</sub>) to water and oxygen.
              </p>
							<p>
								Given its central role in cell processes, catalase is
								taxonomimcally widespread.
								Because it is under strong selective pressure,
								it has a low overall mutation rate –
								making it well suited to tracking deep divergences.
							</p>
              <p>
                We will collect gene sequences from some representative mammals,
                consider appropriate callibration points, and explore the
                sensitivity of our analysis to our assumptions.
              </p>
						</div>
            <div class="col-10 col-12-narrow">

              <h3>Obtain DNA sequences</h3>
              <p>
                We will download sequences of the catalase gene from the
                <acronym title="National Center for Biotechnology information">NCBI</acronym>,
                a database covering the majority of DNA sequences that have been
                collected.
              </p>
              <ol>
                <li>
                  Head to <a href="https://www.ncbi.nlm.nih.gov/gene" target="_blank" title="NCBI gene search">
                https://www.ncbi.nlm.nih.gov/gene</a>.
                </li>
                <li>
                  Type <code>catalase</code> into the search (<code>Gene</code>) box
                </li>
                <li>Click "Search"</li>
								<img class="right" src="images/clock/results-by-taxon.png" />
                <li>
                  Under "Results by Taxon",
									“Top Organisms” displays numbers of sequences from specific taxa.
									click "Tree" to display taxa grouped phylogenetically.
                </li>
								<li>
									Expand animals → chordates → vertebrates → mammals to view
									all available gene sequences.  Notice the taxa for which
									data is available.
								</li>
								<img class="right" src="images/clock/gene-id-cat.png" />
								<li class="clear">
									Find a taxon of your choice with the gene ID "CAT" and
									click the gene ID.

								</li>
								<p>
									Not all similar DNA sequences are necessarily homologous –
									for instance, sometimes a gene is duplicated, and two copies
									evolve independently in the same lineage.
									As mammals are well understood, we can rely on existing
									work to identify sequences of equivalent genes.
								</p>
								<li>
									Scroll down to § "General gene information" and click the link
									"Homology → NCBI Orthologs"
									<a href="https://www.ncbi.nlm.nih.gov/gene/847/ortholog/"
									title="Cut direct to page">[→]</a>.
									This detailed information is only available for vertebrates.
								</li>
								<p>Now let's choose some taxa to study.</p>
								<li>
									Type Marsupials into the "Enter taxonomic name" box and select
									Marsupials (Metatheria) from the dropdown to filter results.
								</li>
								<li>
									You should see half a dozen available sequences.
									Note the taxa for which data are available (the common name
									is given after the binomial epithet), and the length
									of the gene sequences (in amino acids, aa);
									some genes are different lengths.
								</li>
								<li>
									Select four or five taxa and click “Add to cart”.
									Prefer taxa that are included in one of the
									"well-corroborated trees" figured by
									<a href="https://academic.oup.com/view-large/figure/362010550/syab072f1.tif"
									target="_blank">Asher &amp; Smith (2023)</a> –
									it will help to know something about the relationships
									between your chosen taxa.
								</li>
								<li>
									Clear the taxonomic name box and repeat for
									placentals (Eutheria)
								</li>
								<li>
									Now add both the avaialble species of monotreme to your cart.
									The platypus and echidna will together form our outgroup.
								</li>
								<li>
									Open your cart (trolley icon, should read e.g. “12 items”).
								</li>
								<li>
									Select “Protein alignment → one sequence per gene”.
								</li>
								<li>
									Click “Advanced parameters” (under the "Align" button").
								</li>
								<li class="question">Notice the gap opening penalties.
									What do these mean?
									Are the default values sensible?
								</li>
								<li>
									Click "Align" to start computing an alignment.
									This will take a few seconds as the algorithm tries to find
									the best fit match of AA sequences.
								</li>
								<li class="question">
									If the alignment is incorrect,
									what will the consequences be for inference?
								</li>
								<li>
									Observe the graphical overview of the alignment.
									Note the highly conserved nature of the amino acid sequences.
									Scroll down to see the aligned sequences themselves.
								</li>
								<li class="question">
									The monotremes have more amino acids than other mammals.
									Were these extra elements present in the common ancestor?
								</li>
								<p>
									So far we've been aligning the amino acid sequences.
									We could work backwards from this alignment to produce a
									DNA alignment.  But we're instead going to use the mRNA
									nucleotide sequences directly.
								</p>
								<li>
									Close the COBALT alignment tool and return to your Cart.
								</li>
								<li>
									Click "Download" → "RefSeq transcripts (FASTA)"
									(one sequence per gene). This will download the mRNA
									sequences.
								</li>
								<li>
									Open the file in a text editor (I recommend
									<a href="https://notepad-plus-plus.org/downloads/" title="Download Notepad++"
									target="_blank">Notepad++</a>;
									Apple users might prefer
									<a href="https://zed.dev/" target="_blank" title="Zed editor">Zed</a>)
								</li>
								<li>
									Perform a search & replace to replace spaces (<code> </code>)
									with underscores (<code>_</code>).
									<ul>
										<li>
											Optionally, remove other information, to leave only
											the species names – this doesn't affect the analysis,
											but may make your results easier to read.
										</li>
										<li>
											Be sure to retain the initial &ldquo;<code>&gt;</code>&rdquo;,
											such that <code>&gt;NM_001035386.2_Bos_taurus_catalase_(CAT),_mRNA</code>
											becomes <code>&gt;Bos_taurus</code>
										</li>
									</ul>
								</li>
							<h3>Align DNA sequences</h3>
							<p>Now we are ready to align our nucleotide sequences.</p>

								<li>
									Open the <a href="https://www.ebi.ac.uk/jdispatcher/msa/muscle"
									title="MUSCLE multiple sequence alignment" target="_blank">
									MUSCLE web tool</a>
								</li>
								<li>
									Under "Input sequence", select "Choose file" to upload your
									edited FASTA file.
									<ul>
										<li>
											Not working? Try my
											<a href="assets/clock/cat_refseq_transcript.fasta">sample file</a>.
										</li>
									</ul>
								</li>
								<li>
									Under "Parameters", select "Output Format: Pearson/FASTA".
								</li>
								<li>
									Click "Submit" and wait for your results to be ready.
								</li>
								<li>
									Explore your alignment; zoom and drag using the axis.
								</li>
								<img src="images/clock/muscle-alignment.png" />
								<li>
									Notice that the MUSCLE algorithm aligns the sequences assuming
									a phylogenetic explanation for differences; it thus generates
									an <strong>unrooted</strong> phylogenetic tree as part of
									finding the best alignment.
								</li>
								<li class="question">
									How would the tree look different if a root was specified?
								</li>
								<li class="question">
									How do the relationships in your tree compare to those in the
									<a href="https://academic.oup.com/view-large/figure/362010550/syab072f1.tif"
									target="_blank">well-corroborated trees</a>?
								</li>
								<li>
									Download the results in FASTA format by heading to "Tool Output→Download".
									<ul>
										<li>
											If that doesn't work, try "Result Files→Alignment in FASTA format"
										</li>
										<li>
											If necessary, use my
											<a href="assets/clock/cat_aligned.fasta">sample output file</a>.
										</li>
									</ul>
								</li>
								<li>
									Rename the file so it has a <code>.fasta</code> extension
									(replacing the default <code>.txt</code>).
									<ul>
										<li class="bulb">
											Stuck?
											Try asking e.g.
											 <a href="https://bard.google.com/" target="_blank">Bard</a>,
											<a href="https://chat.openai.com/" target="_blank">ChatGPT</a>
											 "How do I change a file extension in Windows 10?"

											 Generative AI often gets to the point faster than top-ranked
											 <a href="https://www.indeed.com/career-advice/career-development/how-to-change-file-type" target="_blank" rel="nofollow">search results</a>,
											 which are often bloated with
											 <acronym title="Search Engine Optimization">SEO</acronym>
											 blather.
										 </li>
									 </ul>
								</li>
								<h3>Design a molecular clock analysis</h3>
								<li>
									If you haven't already done so,
									download BEAST2 and unzip the contents to a folder on your computer.
								</li>
								<li>
									Navigate to this folder and run "BEAUti".
									You may need to approve a security prompt.
								</li>
								<li>
									Import your alignment file using "File→Import alignment"
									<ul>
										<li class="bulb">
											If you see "Unsupported alignment",
										  check that you changed the file extension to <code>.fasta</code>
										</li>
								  </ul>
								</li>
								<img src="images/clock/beauti-import.png" />
								<li>Confirm that the data type is "nucleotide"</li>
								<li>
									<p>
										Open the "Site Model" tab and select a Substitution Model.
									</p>
									<p>
										The default, JC69, assumes a constant mutation rate between
										any of the four nucleotide bases.
									</p>
									<p>
										The HKY model incorporates the fact that chemically similar
										bases are easier to substitute – and tend to encode chemically
										similar amino acids, so mutations are less likely to be
										deleterious.
										The TN93 model fine-tunes this further by adding further
										parameters, whereas the fully generalized GTR
										(Generalized Time Reversible) model allows a distinct mutation
										rate between each different pair of nucleotides (64 in total) –
										whilst potentially more accurate, it takes more data to infer
										the values of all 64 parameters, making the model succeptible
										to overfitting with smaller datasets like ours.
									</p>
									<p>
									 	Select the HKY model and set Frequencies: Empirical.
								 	</p>
								</li>
								<li>
									Open the "Clock Model" tab.  We will use a Strict Clock
									for our initial analysis. Leave the Mean clock rate at its
									default setting (1).
								</li>

								<li>
									Open the "Priors" tab and update the "Tree.t" prior from
									"Yule Model" to "Calibrated Yule Model"
								</li>
								<li>
									Save your progress (File → Save)
								</li>
								<li>
									At the bottom of the "Priors" tab, click "+ Add Prior".
								</li>
								<p>We are going to constrain the age of the common ancestor of a clade.
									Our prior allows us to set a probability distribution on the
									date of the most recent common ancestor
									(<acronym title="most recent common ancestor">MRCA</acronym>)
									of a set of taxa.
								<li>
									Select "MRCA Prior".
									<ul><li class="bulb"><strong>Be careful!</strong>
									Priors cannot be deleted – if you select the wrong one,
									you'll have to start again from your last save file (!)
								</li></ul>
								</li>
								<li>
									Select the taxa in the group you wish to constrain.
									We'll start with the monotremes, <i>Ornithorhynchus</i> and
									<i>Tachyglossus</i>.  Move these taxa to the right-hand
									list (by pressing "&gt;&gt;").
								</li>
								<li>
									Enter "Monotremes" as a taxon set label
								</li>
								<li>
									Click "OK".
								</li>
								<p>
									The prior should now appear on your "Priors" tab.
									At the moment it is an "empty" prior and will not affect
									the analysis.
								</p>
								<li>
									Tick the "Monophyletic" checkbox. This forces these taxa
									to form a clade – and allows us to constrain the date of
									the clade's MRCA.
								</li>
								<li>
									Choose a prior from the the "[none]" drop-down box.
									A "Gamma" prior is probably appropriate here.
								</li>
								<li>
									Select the &#x25B6; button to expand the prior.
									Specify appropriate values for the "Alpha", "Beta" and
									"Offset" parameters.
								</li>
								<img src="images/clock/monotreme-prior.png" />
								<h4>Specifying a prior</h4>
								<p>
									This is a good point to ask what an appropriate prior
									distribution might look like.
									Bayesian priors attempt to encapuslate our prior beliefs,
									i.e. what we think we know before we run the analysis.
								</p>
								<p>
									At this point you probably don't have much prior
									knowledge about monotreme origination.  But such information
									is seldom far away; a Google Scholar search for
									“<a href="https://scholar.google.co.uk/scholar?q=Oldest+monotreme" target="_blank">
									Oldest monotreme</a>” throws up a 2008 PNAS article
									(fourth result for me)
									“<a href="https://dx.doi.org/10.1073/pnas.0706385105" title="The oldest platypus and its bearing on divergence timing
									of the platypus and echidna clades" target="_blanks">
									The oldest platypus and its bearing on divergence timing
									of the platypus and echidna clades</a>”.
								</p>
								<p>
									This study is perfect for our needs:
									it reports a total-group platypus, which is by definition a
									crown-group member of the clade we have constrained
									(strictly, Platypus + Echidna – there may be other crown-group
									monotremes not included in our analysis).
									This fossil is dated to 121–112.5&nbsp;Ma, so the MRCA of
									platypus and echidna <strong>must</strong> be older than 112.5&nbsp;Ma;
									this gives us a value for our "Offset" parameter.
								</p>
								<p>
									The other parameters are more subjective.
									This this is a rather derived fossil and
									(based on the PNAS abstract and the snippets in the other Google Scholar results)
									the fossil record of monotremes seems to be quite sparse,
									so I think it’s quite likely that there are older fossils
									remaining to be discovered – it would be surprising if the
									only relevant fossil we'd found was so close in time to the MRCA.
								</p>
								<p>
									The Beta parameter allows us to scale ("stretch") the distribution.
									The default value of 2 seems too small for me;
									our default priors assign a 95% probability to the MRCA
									being younger than 122&nbsp;Ma (see the Quantile report under
									the curve visualization), but our dating uncertainty on the
									fossil itself spreads back to 121&nbsp;Ma.
									Let's try a larger value: perhaps 200?
								</p>
								<p>
									Does this prior look reasonable?  Check the shape of the
									curve and its summary statistics.
									The very earliest stem-group mammal fossils are ~320&nbsp;Ma,
									so it's highly unlikely that the
									platupus-echidna MRCA dates back that far –
									The mammal crown group is probably not older than
									Morganucodonts, late Triassic (205&nbsp;Ma) upper-stem Mammals,
									so we probably don’t want a prior that assigns much probability
									to ages older than 205&nbsp;Ma for our monotreme divergence node.
									Perhaps a Beta value of 20 looks more reasonable?
								</p>
								<p>
									We should also consider that the fossil is quite derived,
									and that the ~30&nbsp;Ma gap to the second-oldest platypus
									fossil suggests that the monotreme fossil record is rather
									incomplete.
									These considerations can be reflected in our Alpha parameter.
									Compare values of 1 and 10 to get a feeling for how
									this parameter changes the shape of the probability
									distribution.
									If we felt it was likely that our constraining fossil was
									very close to the date of the MRCA, we might pick a small
									value of alpha; in this case, I am inclined to opt for a
									larger value of Alpha, which increases the mean of the
									distribution – simply put, it moves the "hump" of the
									distribution further back in time.
								</p>
								<p>
									Increasing Alpha also extends the tail of the distribution,
									so Beta will need adjusting accordingly: I'm happy with values
									of Alpha = 3; Beta = 10; Offset = 112.5.
								</p>
								<p>
									A vertebrate palaeontologist with a better understanding
									of platypus morphology and mammal taphonomy would no doubt
									come up with a different set of values.
									If we have lots of informative data, we hope that the signal
									will outweigh the prior: we update our beliefs in light
									of new evidence.
								  But if our data are not particularly informative, we might
									find that our results simply echo our priors back to us:
									in which case, justifying the precise shape of our prior
									distribution becomes even more important.
								</p>
								<li>
									Select at least one, ideally two or three, further clades to
									constrain.  Search the literature to justify your design
									of priors.
								</li>
								<li>
									To root the tree, add an MRCA prior including everything that's
									not a monotreme. Label it appropriately
									("Theria", or simply "ingroup").
									Leave the prior type to "[none]" and select "monophyletic".
									This will force the tree to be rooted on monotremes,
									and produce a summary for the posterior estimate of the
									date of the MRCA of placentals and marsupuals (the therians).
								</li>
								<li>
									Finally, configure the
									<acronym title="Monte Carlo Markov Chain">MCMC</acronym>
									analysis.  Set the Chain Length to 100&nbsp;000.
									Simply put, this will try 100k different parameter variations
									(e.g. mutation rate, tree topology, branch lengths),
									sampling more plausible values of the parameters with higher
									frequencies.
								</li>
								<li>
									Safe your settings (File→Save).
								</li>
								<h3>Conduct the analysis</h3>
								<li>Return to your BEAST folder and run BEAST</li>
								<li>
									Select the "Input file" you just created in BEAUti.
									<ul><li class="bulb">
										Work from a <a href="assets/clock.cat_initial.xml">template</a>
										if necessary
									</li></ul>
								</li>
								<li>Notice the box reading "default: only write new log files".
									<ul><li class="bulb">
										Log files will be named based on the input file;
										if you wish to overwrite or add to a previous run, select
										the relevant option, otherwise the presence of previous log
										files will cause the run to abort with an error.
									</li></ul>
								</li>
								<li>Press "Run" to begin the analysis.</li>
								<li>
									Open Tracer
								</li>
								<li>
									Under "Trace Files:", press "+" to load your BEAST log.
									<ul><li class="bulb">
										If the run's not finished,
										Tracer will display the current results; press the "Reload"
										button to update.
									</li>
									<li>
										Use a <a href="assets/clock/cat_initial.log">template file</a>
										if needed.
									</li></ul>
								</li>
								<li>
									Note the Estimated Sample Size for each parameter.
									This is an indication of how well the range of possible values
									has been explored; values &lt; 200 are not likely to be
									statistically robust estimates.

							</ol>



              <h3>Under construction</h3>
              <p>The rest of this content is on its way.</p>

              <p>Meanwhile, you might find a primer on
              <a href="https://www.nature.com/scitable/topicpage/the-molecular-clock-and-estimating-species-divergence-41971/" target="_blank">molecular clocks</a>
              useful.
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>

    <!--section id="q-and-a" class="main">
      <header>
        <div class="container">
          <span class="feature-icon">
            <span class="icon solid fa-question"></span>
          </span>
          <h2>Questions</h2>
        </div>
      </header>
      <div class="content">
        <div class="container">
          <p>
            Propose and vote for questions or topics to cover during the
            face-to-face session.
          </p>
          <iframe src="https://app.sli.do/event/nQyN7PoeaWTDJp8QuAWwCW/questions"
                  height="100%" width="100%" style="min-height: 560px;">Loading…</iframe>
        </div>
      </div>
    </section-->

		<!-- Footer -->
    <section id="footer">
      <div class="copyright">
        <ul class="menu">
          <li>&copy; 2024 <a href="http://smithlabdurham.github.io/">Martin R. Smith</a></li>
          <li>All content is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>, except where noted.</li>
          <li>Design: <a href="http://html5up.net" rel='noopener'>HTML5 UP</a></li>
        </ul>
      </div>
    </section>

		<!-- Scripts -->
    <script src="assets/js/jquery.min.js"></script>
    <script src="assets/js/jquery.scrolly.min.js"></script>
    <script src="assets/js/browser.min.js"></script>
    <script src="assets/js/breakpoints.min.js"></script>
    <script src="assets/js/util.js"></script>
    <script defer src="assets/js/thisyear.js"></script>
    <script defer src="assets/js/main.js"></script>
	</body>
</html>
