<!DOCTYPE HTML>
<!--
	Template: Tessellate by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>GEOL2301: Frontiers in palaeontology</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />

    <link rel="apple-touch-icon" sizes="180x180" href="icons/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="icons/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="icons/favicon-16x16.png">
    <link rel="manifest" href="icons/site.webmanifest">
    <link rel="mask-icon" href="icons/safari-pinned-tab.svg" color="#5bbad5">
    <link rel="shortcut icon" href="icons/favicon.ico">
    <meta name="msapplication-TileColor" content="#da532c">
    <meta name="msapplication-config" content="icons/browserconfig.xml">
    <meta name="theme-color" content="#ffffff">
	</head>
	<body class="is-preload">

		<!-- Header -->
    <section id="header" class="dark">
      <header>
        <h1>GEOL2301: Frontiers in palaeontology</h1>
        <p>Session 4: Molecular clocks</p>
      </header>
      <footer>
        <a href="#first" class="button scrolly">Begin</a>
      </footer>
    </section>
    
    
    <!-- First -->
    <section id="first" class="main">
      <header>
        <div class="container">
          <h2>Molecular clocks</h2>
          <p>
            This week we&rsquo;ll explore the journey from DNA sequences to 
            dated phylogenies by performing our own molecular clock analyses.
          </p>
          <p>
            Before you start, download and unpack
            <a href="https://www.beast2.org/" target="_blank" title="Download BEAST">BEAST2</a>,
            <a href="https://github.com/beast-dev/tracer/releases/tag/v1.7.2" title="Download Tracer" target="_blank">Tracer</a>,
            and (Windows users only) 
            <a href="https://www.java.com/en/" target="_blank" title="Download Java Runtime Environment">Java</a>.
            <br />
            (For Tracer, Look under “Assets” for the release appropriate to your computer:
            Windows = <code>Tracer.zip</code>; Mac = <code>.dmg</code>)  
          </p>            
        </div>
      </header>
      <div class="content dark style1 featured">
        <div class="container">
          <div class="row">
            <div class="col-4 col-12-narrow">              
              <span class="feature-icon"><span class="icon solid fa-dice-one"></span></span>
              <p>Understand what controls the accumulation of mutations in species&rsquo; DNA</p>
            </div>
            <div class="col-4 col-12-narrow">
              <span class="feature-icon"><span class="icon solid fa-dice-two"></span></span>
              <p>Perform a molecular clock analysis</p>
            </div>
            <div class="col-4 col-12-narrow">
              <span class="feature-icon"><span class="icon solid fa-dice-three"></span></span>
              <p>Evaluate the assumptions and limitations of molecular clock results</p>
            </div>
            <div class="col-12">
              <footer>
                <a href="#second" class="button scrolly">Let&rsquo;s get started!</a>
              </footer>
            </div>
          </div>
        </div>
      </div>
    </section>
    
  <!-- Second -->
    <section id="second" class="main">
      <header>
        <div class="container">
          <h2>Molecular sequence data</h2>
          <p>
            From DNA sequences to dated phylogenies
          </p>
        </div>
      </header>
      <div class="content dark style1">
        <div class="container">
          <div class="row">
            <div class="col-8 col-12-narrow">
              <h3>How DNA makes proteins</h3>
              <p>
                DNA comprises a backbone of the sugar deoxyribose that holds
                in place a series of nucleotide <strong>bases</strong>,
                typically referred to by their initial letters:
                adenine (A), cytosine (C), guanine (G) and thymine (T).
              </p>
              
              <p>
                A sequence of three bases forms a <strong>codon</strong>, which
                encodes a specific amino acid.
                Notice that the base in the third position of a codon is often
                redundant – this base can often be changed without affecting
                the translation.
              </p>
              <img src="images/clock/Codon-Chart-2.svg" />
              
              <p>
                Some sections of DNA encode <strong>genes</strong>.
                In animals, genes start with an <code>ATG</code> codon, and end 
                with one of three "STOP" codons.
                An enzyme, RNA polymerase, binds to the start codon (perhaps
                with the help of transcription factors that bind to the DNA
                sequence before the start of the gene) and makes
                messenger RNA copies of the nucleotide sequence.
              </p>
              <p>
                <strong>Ribosomes</strong> then bind to the start of the mRNA 
                sequences and work their way along.
                At each codon position, they bind the corresponding amino acid
                to a growing chain.  When they reach the STOP codon at the end
                of the mRNA sequence, the completed amino acid sequence folds
                to form a protein, often with a complex three-dimensional
                structure.
              </p>
              
              <p>
                Mutations cause changes to individual nucleotide bases within
                a DNA sequence – or may cause bases to be added or lost.
                What impact might mutations have on the resulting protein?
              </p>
              
              <ul>
                <li class="question" onclick="Reveal('mutate-base');">Changing an individual base</li>
                <li class="bulb">Think about the specific codons <code>TTG</code> and <code>TGT</code></li>
                <li class="hidden written answer" id="mutate-base">
                  <p>If the base is in the third codon position, often there will be no effect.
                  Some first-position changes have no effect, too.
                  </p><p>
                  Changes in the second position usually change the amino acid. 
                  Amino acids with similar chemical properties often share similar codes – 
                  but a change in an amino acid in the active site of an enzyme could
                  render the protien non-functional.
                  </p><p>
                  Cystine (Cys) contains sulfur and forms strong S-S bonds that often 
                  control how a protein folds; removing these structural elements may be
                  more catastrophic than swapping two amino acids with similar chemistry.
                  </p>
                </li>
                <li class="question" onclick="Reveal('add-base');">Adding a new base</li>
                <li class="hidden written answer" id="add-base">
                  This will shift the reading frame; every codon after the insertion will encode a different amino acid.
                  This is likely disastrous for the protein, as the sequence of amino acids will be essentially random.
                </li>
                <li class="question" onclick="Reveal('lose-bases');">Losing a series of bases</li>
                <li class="hidden written answer" id="lose-bases">If the number of bases lost is a multiple of three, the odds of a functional output are higher, as the reading frame is not shifted.</li>
              </ul>
              
              <p>
                Deleterious mutations – those that decrease the fitness of an 
                individual – are unlikely to persist in the gene pool.
                Hence the rate at which (germ line) mutations occur is higher
                than the rate at which <em>persistent</em> mutations are added
                to the gene pool of a population or species.
              </p>
              
              <p>
                If this latter rate is approximately uniform over time, we might
                in principle use it to deduce how recently two lineages diverged.
              </p>
              
              <ul>
                <li class="question" onclick="Reveal('mol-calc');">
                  Assume that one mutation modifies a particular gene's sequence
                  every million years.  Two species have sequences that
                  differ in 50 places.  Based on a quick calculation, estimate 
                  how many years may have elapsed since their last common ancestor.
                </li>
                <li class="bulb">It may help to draw a simple phylogenetic tree</li>
                <li class="hidden written answer" id="mol-calc">
                  <p>
                  A good first guess might be 25&nbsp;million years.
                  The two species each followed separate evolutionary pathways
                  for 25&nbsp;Ma; the <strong>ancestral</strong> sequence would
                  differ from each living descendant in 25 places.
                  </p>
                </li>
                <li class="question" onclick="Reveal('mol-more');">
                  Why might the true divergence date be higher than a simple
                  calculation would suggest?
                </li>
                <li class="bulb">
                  Consider a case where the common ancestor had an
                  <code>A</code> at a specific position.
                  Think of scenarios where mutations may have occurred, but
                  not be observable based on the sequences available.
                </li>
                <li class="hidden written answer" id="mol-more">
                  <p>
                    If our ancestor has <code>A</code>:
                    <ul>
                      <li>
                        One descendant may have mutated to <code>C</code>,
                        then mutated again to <code>G</code>.
                        We'd count one mutation where two had occurred.
                      </li>
                      <li>
                        One descendant may have mutated to <code>C</code>,
                        then mutated again to the original <code>A</code>.
                        We'd count no mutations, where two had occurred.
                      </li>
                      <li>
                        One descendant might have mutated to <code>C</code>,
                        the other to <code>G</code>.
                        We'd count one mutation where two had occurred.
                      </li>
                      <li>
                        Both descendants might have mutated to <code>C</code>.
                        We'd count no mutations, where two had occurred.
                      </li>
                    </ul>
                  </p>
                </li>

                <li class="question" onclick="Reveal('mol-const');">
                  How confident do you feel that we can assume a single, constant
                  mutation rate across all taxa?
                </li>
                <li class="hidden written answer" id="mol-const">
                  Many factors might work against this – population size; 
                  generation time (i.e. opportunities for mutations to occur);
                  plasticity of organisms; whether taxa are generalists or specialists
                  (likelihood that deleterious mutations might be overcome);
                  mass extinctions or evolutionary radiations 
                  (new opportunities may change selection constraints).
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </section>
    
    <!-- Third -->
    <section id="third" class="main">
      <header>
        <div class="container">
          <h2>A first molecular clock analysis</h2>
          <p>
            Conducting a simple analysis to explore the operation of the molecular clock
          </p>
        </div>
      </header>
      <div class="content dark style1">
        <div class="container">
          <div class="row">
            <div class="col-8 col-12-narrow">
              
              <h3>Under construction</h3>
              <p>The rest of this content is on its way.</p>
              
              <p>Meanwhile, you might find a primer on 
              <a href="https://www.nature.com/scitable/topicpage/the-molecular-clock-and-estimating-species-divergence-41971/" target="_blank">molecular clocks</a>
              useful.
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>

    <section id="q-and-a" class="main">
      <header>
        <div class="container">
          <span class="feature-icon">
            <span class="icon solid fa-question"></span>
          </span>
          <h2>Questions</h2>
        </div>
      </header>
      <div class="content">
        <div class="container">
          <p>
            Propose and vote for questions or topics to cover during the
            face-to-face session.
          </p>
          <iframe src="https://app.sli.do/event/nQyN7PoeaWTDJp8QuAWwCW/questions"
                  height="100%" width="100%" style="min-height: 560px;">Loading…</iframe>
        </div>
      </div>
    </section>

		<!-- Footer -->
    <section id="footer">
      <div class="copyright">
        <ul class="menu">
          <li>&copy; 2024 <a href="http://smithlabdurham.github.io/">Martin R. Smith</a></li>
          <li>All content is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>, except where noted.</li>
          <li>Design: <a href="http://html5up.net" rel='noopener'>HTML5 UP</a></li>
        </ul>
      </div>
    </section>
    
		<!-- Scripts -->
    <script src="assets/js/jquery.min.js"></script>
    <script src="assets/js/jquery.scrolly.min.js"></script>
    <script src="assets/js/browser.min.js"></script>
    <script src="assets/js/breakpoints.min.js"></script>
    <script src="assets/js/util.js"></script>
    <script defer src="assets/js/thisyear.js"></script>
    <script defer src="assets/js/main.js"></script>
	</body>
</html>