<!DOCTYPE HTML>
<!--
	Template: Tessellate by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>GEOL2317: Frontiers in palaeontology</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />

    <link rel="apple-touch-icon" sizes="180x180" href="icons/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="icons/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="icons/favicon-16x16.png">
    <link rel="manifest" href="icons/site.webmanifest">
    <link rel="mask-icon" href="icons/safari-pinned-tab.svg" color="#5bbad5">
    <link rel="shortcut icon" href="icons/favicon.ico">
    <meta name="msapplication-TileColor" content="#da532c">
    <meta name="msapplication-config" content="icons/browserconfig.xml">
    <meta name="theme-color" content="#ffffff">
	</head>
	<body class="is-preload">

		<!-- Header -->
    <section id="header" class="dark">
      <header>
        <h1>GEOL2317: Frontiers in palaeontology</h1>
        <p>Session 4: Molecular clocks</p>
      </header>
      <footer>
        <a href="#first" class="button scrolly">Begin</a>
      </footer>
    </section>

          
    <!-- First -->
    <section id="first" class="main">
      <header>
        <div class="container">
          <h2>Molecular clocks</h2>
          <p>
            This week we&rsquo;ll explore the journey from DNA sequences to
            dated phylogenies by performing our own molecular clock analyses.
          </p>
          <p>
            Before you start, download and unpack
            <a href="https://www.beast2.org/" target="_blank" title="Download BEAST">BEAST2</a>,
            <a href="https://github.com/beast-dev/tracer/releases/tag/v1.7.2" title="Download Tracer" target="_blank">Tracer</a>,
            and (Windows users only)
            <a href="https://www.java.com/en/" target="_blank" title="Download Java Runtime Environment">Java</a>.
            <br />
            (For Tracer, Look under “Assets” for the release appropriate to your computer:
            Windows = <code>Tracer.zip</code>; Mac = <code>.dmg</code>)
          </p>
        </div>
      </header>
      <div class="content dark style1 featured">
        <div class="container">
          <div class="row">
            <div class="col-4 col-12-narrow">
              <span class="feature-icon"><span class="icon solid fa-dice-one"></span></span>
              <p>Understand what controls the accumulation of mutations in species&rsquo; DNA</p>
            </div>
            <div class="col-4 col-12-narrow">
              <span class="feature-icon"><span class="icon solid fa-dice-two"></span></span>
              <p>Perform a molecular clock analysis</p>
            </div>
            <div class="col-4 col-12-narrow">
              <span class="feature-icon"><span class="icon solid fa-dice-three"></span></span>
              <p>Evaluate the assumptions and limitations of molecular clock results</p>
            </div>
            <div class="col-12">
              <footer>
                <a href="#second" class="button scrolly">Let&rsquo;s get started!</a>
              </footer>
            </div>
          </div>
        </div>
      </div>
    </section>

  <!-- Second -->
    <section id="second" class="main">
      <header>
        <div class="container">
          <h2>Molecular sequence data</h2>
          <p>
            From DNA sequences to dated phylogenies
          </p>
        </div>
      </header>
      <div class="content dark style1">
        <div class="container">
          <div class="row">
            <div class="col-8 col-12-narrow">
              <h3>How DNA makes proteins</h3>
              <p>
                DNA, bundled in the chromosomes within the nucleus of each cell,
                comprises a backbone of the sugar deoxyribose that holds
                in place a series of nucleotide <strong>bases</strong>,
                typically referred to by their initial letters:
                adenine (A), cytosine (C), guanine (G) and thymine (T).
              </p>

              <p>
                A sequence of three bases forms a <strong>codon</strong>, which
                encodes a specific amino acid.
                Notice that the base in the third position of a codon is often
                redundant – this base can often be changed without affecting
                the translation.
              </p>
              <img src="images/clock/Codon-Chart-2.svg" />

              <p>
                Some sections of DNA encode <strong>genes</strong>.
                In animals, genes start with an <code>ATG</code> codon, and end
                with one of three "STOP" codons.
                An enzyme, RNA polymerase, binds to the start codon (perhaps
                with the help of transcription factors that bind to the DNA
                sequence before the start of the gene) and makes
                messenger RNA copies of the nucleotide sequence.
              </p>
              <p>
                <strong>Ribosomes</strong> then bind to the start of the mRNA
                sequences and work their way along.
                At each codon position, they bind the corresponding amino acid
                to a growing chain.  When they reach the STOP codon at the end
                of the mRNA sequence, the completed amino acid sequence folds
                to form a protein, often with a complex three-dimensional
                structure.
              </p>

              <p>
                Mutations cause changes to individual nucleotide bases within
                a DNA sequence – or may cause bases to be added or lost.
                What impact might mutations have on the resulting protein?
              </p>

              <ul>
                <li class="question" onclick="Reveal('mutate-base');">Changing an individual base</li>
                <li class="bulb">Think about the specific codons <code>TTG</code> and <code>TGT</code></li>
                <li class="hidden written answer" id="mutate-base">
                  <p>If the base is in the third codon position, often there will be no effect.
                  Some first-position changes have no effect, too.
                  </p><p>
                  Changes in the second position usually change the amino acid.
                  Amino acids with similar chemical properties often share similar codes –
                  but a change in an amino acid in the active site of an enzyme could
                  render the protien non-functional.
                  </p><p>
                  Cystine (Cys) contains sulfur and forms strong S-S bonds that often
                  control how a protein folds; removing these structural elements may be
                  more catastrophic than swapping two amino acids with similar chemistry.
                  </p>
                </li>
                <li class="question" onclick="Reveal('add-base');">Adding a new base</li>
                <li class="hidden written answer" id="add-base">
                  This will shift the reading frame; every codon after the insertion will encode a different amino acid.
                  This is likely disastrous for the protein, as the sequence of amino acids will be essentially random.
                </li>
                <li class="question" onclick="Reveal('lose-bases');">Losing a series of bases</li>
                <li class="hidden written answer" id="lose-bases">If the number of bases lost is a multiple of three, the odds of a functional output are higher, as the reading frame is not shifted.</li>
              </ul>

              <p>
                Deleterious mutations – those that decrease the fitness of an
                individual – are unlikely to persist in the gene pool.
                Hence the rate at which (germ line) mutations occur is higher
                than the rate at which <em>persistent</em> mutations are added
                to the gene pool of a population or species.
              </p>

              <p>
                If this latter rate is approximately uniform over time, we might
                in principle use it to deduce how recently two lineages diverged.
              </p>

              <ul>
                <li class="question" onclick="Reveal('mol-calc');">
                  Assume that one mutation modifies a particular gene's sequence
                  every million years.  Two species have sequences that
                  differ in 50 places.  Based on a quick calculation, estimate
                  how many years may have elapsed since their last common ancestor.
                </li>
                <li class="bulb">It may help to draw a simple phylogenetic tree</li>
                <li class="hidden written answer" id="mol-calc">
                  <p>
                  A good first guess might be 25&nbsp;million years.
                  The two species each followed separate evolutionary pathways
                  for 25&nbsp;Ma; the <strong>ancestral</strong> sequence would
                  differ from each living descendant in 25 places.
                  </p>
                </li>
                <li class="question" onclick="Reveal('mol-more');">
                  Why might the true divergence date be higher than a simple
                  calculation would suggest?
                </li>
                <li class="bulb">
                  Consider a case where the common ancestor had an
                  <code>A</code> at a specific position.
                  Think of scenarios where mutations may have occurred, but
                  not be observable based on the sequences available.
                </li>
                <li class="hidden written answer" id="mol-more">
                  <p>
                    If our ancestor has <code>A</code>:
                    <ul>
                      <li>
                        One descendant may have mutated to <code>C</code>,
                        then mutated again to <code>G</code>.
                        We'd count one mutation where two had occurred.
                      </li>
                      <li>
                        One descendant may have mutated to <code>C</code>,
                        then mutated again to the original <code>A</code>.
                        We'd count no mutations, where two had occurred.
                      </li>
                      <li>
                        One descendant might have mutated to <code>C</code>,
                        the other to <code>G</code>.
                        We'd count one mutation where two had occurred.
                      </li>
                      <li>
                        Both descendants might have mutated to <code>C</code>.
                        We'd count no mutations, where two had occurred.
                      </li>
                    </ul>
                  </p>
                </li>

                <li class="question" onclick="Reveal('mol-const');">
                  How confident do you feel that we can assume a single, constant
                  mutation rate across all taxa? (Why?)
                </li>
                <li class="hidden written answer" id="mol-const">
                  Many factors might work against this – population size;
                  generation time (i.e. opportunities for mutations to occur);
                  plasticity of organisms; whether taxa are generalists or specialists
                  (likelihood that deleterious mutations might be overcome);
                  mass extinctions or evolutionary radiations
                  (new opportunities may change selection constraints).
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </section>

		  <!-- Second -->
		    <section id="second" class="main">
		      <header>
		        <div class="container">
		          <h2>Probabilistic thinking</h2>
		          <p>
		            Key concepts in Bayesian statistics
		          </p>
		        </div>
		      </header>
		      <div class="content dark style1">
		        <div class="container">
		          <div class="row">
		            <div class="col-8 col-12-narrow">
		              <h3>Priors and posteriors</h3>
											  
						  
							<p>
								<em>Frequentist</em> statistics underpin familiar tests, such
								as T-tests or tests for statistical significance.
								Frequentist statistics seeks to estimate the likelihood
								of the <em>data</em> on the assumption that a specified
								hypothesis (or model) is true.
							</p>
							<p>
								Ultimately, we can only draw conclusions indirectly from this
								approach – we're not really interested in the probability of
								the data, because once we have observed the data, we know what
								the data are!
							</p>
							<p>
								<em>Bayesian</em> probability is a statistical paradigm that
								aims to
								estimate the probability of a hypothesis (or model) based on
								available data – which is a much more useful thing to know.
							</p>
							<p>
								The drawback here is that whilst data can make a particular
								hypothesis more or less likely, data alone cannot provide
								an <em>absolute</em> value of the probability of a hypothesis;
								they can only <em>update</em> probabilities.
								Without getting into the maths, the workflow goes like this:
							</p>
							<blockquote>
								Initial probability + data → Improved estimate of probability
							</blockquote>
							<p>
								This initial probability estimate is termed a "prior"
								probability: it sums up our beliefs before we had observed any
								data.
								Critics of a Bayesian approach contend that this is a subjective
								step; the counterargument is that with enough data, the choice
								of prior should have very little influcence on the final
								estimate of probability.
							</p>
						</div>
					
            <div class="col-4 col-12-narrow">
              <div class="resources">
                <h4>
                  Awesome Resources
                </h4>
                <ul>
                  A light introduction to Bayesian thinking:
                  <li>
                    <a href="
https://www.countbayesie.com/blog/2015/2/18/hans-solo-and-bayesian-priors">Han Solo and Bayesian Priors</a>
                  </li>
                </ul>
              </div>
            </div>
					</div>
		          <div class="row">
		            <div class="col-8 col-12-narrow">
									<h4>Bent coin example</h4>
									<p>
										Let's take an example to see how this works in practice.
									</p>
									<p>
										I'm going to toss a coin a few times, then offer a generous
										wager: you give me a pound if the next toss shows heads;
										I give you two pounds if it's tails.
									</p>
									<p>
										We'll consider two students.
									</p>
									<ul>
										<li>
											Cynical Cyril smells a rat, and is 99% sure that I'm using a
											bent coin that's biased to come down heads.
											Based on his knowledge of coin-bending, a bent coin would land
											"heads" in 90% of tosses.
										</li>
										<li>
											Naïve Niamh trustingly assumes that I will use a fair coin,
											whilst acknowledging that there's a 1% chance that I'm
											cheating with a bent coin.
										</li>
									</ul>
									<p>
										The students are thus evaluating the probability of two
										hypotheses: (i), a fair coin that shows heads 50% of the time;
										and (ii), a bent coin that shows heads 90% of the time.
										They'll only accept my wager if there's a good chance that
										hypothesis (i) is true.
									</p>
									<p>
										Let's see what happens when we toss the coin.
										The first toss comes down "heads". What have we learned?
										Well, a single toss does not give us much data at all,
										and our "posterior" odds – the probabilities of the
										two hypotheses <em>after</em> having seen the data – are
										probably very similar to the prior odds.
									</p>
									<p>
										Now, the next three tosses come up heads too; we have
										four heads in total.  This outcome is quite unlikely under
										hypothesis (i) – only one run of tosses in 16 would give four
										heads.  It's much more in line with what we'd suspect under
										hypothesis (ii).  As such, Bayesian analysis would update the
										priors to give more weight to the bent coin hypothesis;
										after seeing the data, Cyril's hunch that I'm using a bent coin
										will be reinforced, whilst Niamh will start pondering whether
										her initial judgement was too trusting.
										Nonetheless, after just four tosses, Naimh will still
										place a much higher probability on the "fair coin" hypothesis
										than Cyril would.
									</p>
									<p>
										Eventually, after enough tosses, we have so much data that the
										choice of priors becomes irrelevant: if 2&nbsp;000 tosses reveal
										a thousand heads and a thousand tails, the "fair coin"
										hypothesis would receive a posterior probability close to one
										under all but the most extreme priors.
									</p>
								</div>
									<div class="col-4 col-12-narrow">
										<span class="image whitepad">
			              	<img src="images/clock/heads.webp" alt="Credit: Royal Mint">
				            </span>
									</div>
								</div>
								<div class="row">
									<div class="col-8 col-12-narrow">
									<h4>Reflecting on the model</h4>
									<p>
										It might be reasonable here to question our underlying model:
										it is probably a simplification to contend that a coin can
										shows heads either 50% or 90% of the time (but nothing in
										between).
									</p>
									<p>
										Probably it is more reasonable to image that a coin, when
										tossed, lands "heads" with a probability
										<acronym title="probability">P</acronym>(<acronym title="Heads">H</acronym>).
										P(H) is a <em>parameter</em> of the model, whose value we are
										interested in estimating. Cyril's prior places zero probability
										on values of P(H) other than 50% and 90%.
										But what if the bias was more subtle, and the coin really landed
										heads just 60% of the time? Cyril would reach the wrong
										conclusion; because probabilities are multiplicative,
										no amount of data can overcome his implicit 0% prior on
										P(H) = 60%.
									</p>
									<p>
										We may thus wish to imagine more flexible priors.
										We could manually assign some prior probability to a range of
										outcomes: perhaps we belive there's a 50% chance that the coin
										is basically fair, landing heads 50–60% of the time;
										a 20% chance it's slightly bent (60–75% heads);
										a 20% chance it's strongly bent (75%–90%),
										and a 10% chance it's extremely bent (90–100% heads).
									</p>
									<p>
										We could go further still and use a continuous distribution
										to divide probability: we could use
										a uniform prior that assigns equal prior probability to all
										values of P(H) between 0% and 100%;
										or some bell-shaped probability distribution curve centred on 50%.
									</p>
									<p>
										With no data, our posterior estimate of the probability of
										"heads" will be will be identical to our prior; but as data
										comes in, our posterior estimate will gradually morph from
										the shape of the prior towards a tight peak centred on the
										true value of P(H).
									</p>
									<p>
										In summary, to conduct a Bayesian analysis, we need:
										<ul>
											<li>
												A model that accounts for our data – here, this encapsulates
												things like the the possible outcomes
												(a coin can be heads or tails but nothing else)
												and the assumption that successive outcomes are independent;
											</li>
											<li>
												Priors: an initial estimate of the probability distribution
												associated with each parameter of the model, whose impact
												diminishes as data accumulate.
											</li>
										</ul>
									</p>
								</div>
								<div class="col-4 col-12-narrow">
									<span class="image whitepad">
										<img src="images/clock/bent_coin.jpg"
										 alt="Credit: Kate Sumnall, Museum of London"
										origin="https://commons.wikimedia.org/wiki/File:A_post_medieval_silver_sixpence_of_Elizabeth_I_%28AD1558-1603%29_dated_AD1571_bent,_probably_to_form_a_love_token._%28FindID_600511%29.jpg">
			            </span>
								</div>
							</div>
						</div>
					</div>
				</section>

    <!-- Fourth -->
    <section id="fourth" class="main">
      <header>
        <div class="container">
          <h2>A first molecular clock analysis</h2>
          <p>
            Conducting a simple analysis to explore the operation of the molecular clock
          </p>
        </div>
      </header>
      <div class="content dark style1">
        <div class="container">
          <div class="row">
            <div class="col-8 col-12-narrow">
              <h3>Objectives</h3>
              <p>
                We will conduct a simple molecular clock to estimate the timing
                of mammal origins, using sequences of the gene catalase –
                a core cellular enzyme responsible for breaking down the
                "free radical" (oxidative stressor) hydrogen peroxide
                (H&#x2082;O&#x2082;) to
								water (H&#x2082;O) and
								oxygen (O&#x2082;) .
              </p>
							<p>
								Given its central role in cell processes, catalase is
								taxonomimcally widespread.
								Because it is under strong selective pressure,
								it has a low overall mutation rate –
								making it well suited to tracking deep divergences
								(e.g.
								<a href="https://dx.doi.org/10.1073/pnas.0401670101"
								title="PNAS: Estimating metazoan divergence times with a molecular clock"
								target="_blank">Petersen <i>et al.</i> 2004</a>).
							</p>
              <p>
                We will collect gene sequences from some representative mammals,
                consider appropriate callibration points, and explore the
                sensitivity of our analysis to our assumptions.
              </p>
						</div>
            <div class="col-10 col-12-narrow">

              <h3>Obtain DNA sequences</h3>
              <p>
                We will download sequences of the catalase gene from the
                <acronym title="National Center for Biotechnology information">NCBI</acronym>,
                a database covering the majority of recorded DNA sequences.
              </p>
              <ol>
                <li>
                  Head to the NCBI Ortholog page for <a href="https://www.ncbi.nlm.nih.gov/gene/847/ortholog/"
									title="Cut direct to page">catalase</a>.
								</li>
								<li>
                  Select some taxa to study.
                  Start by typing <i>marsupials</i> into the "Enter taxonomic name" box.
                  Select
									<i>Marsupials (Metatheria)</i> from the dropdown to filter results.
								</li>
								<li>
									You should see several sequences.
									Note the taxa for which data are available (the common name
									is given after the binomial epithet), and the slight variability in the length
									of the catalase gene (in amino acids, aa).
								</li>
								<li>
									Select four or five taxa and click “Add to cart”.
									The table outlines marsupial relationships; each order is
									monophyletic, and the Australian
									Australidelphia form a clade within Australidelphia.
								</li>
								<table class="middle">
									<thead>
										<th>Superorder</th>
										<th>Order</th>
										<th>Genus</th>
										<th>Common name</th>
										<th></th>
									</thead>
									<tbody>
										<tr>
											<td>—</td>
											<td>Didelphimorphoa</td>
											<td><i>Monodelphis</i></td>
											<td>Gray short-tailed opossum</td>
											<td><img class="silhouette" src="images/clock/phylopic/Monodelphis.svg" /></td>
										</tr>
										</th>
										<tr>
											<td>—</td>
											<td>Didelphimorphoa</td>
											<td><i>Gracilinanus</i></td>
											<td>Gracile mouse opossum</td>
											<td><img class="silhouette" src="images/clock/phylopic/Gracilinanus.svg" /></td>
										</tr>
										</th>
										<tr>
											<td>Australidelphia</td>
											<td>Microbiotheria</td>
											<td><i>Dromiciops</i></td>
											<td>Monito del monte</td>
											<td><img class="silhouette" src="images/clock/phylopic/Dromiciops.svg" /></td>
										</tr>
										<tr>
											<td>Australidelphia (Australian)</td>
											<td>Diprotodontia</td>
											<td><i>Vombatus</i></td>
											<td>Wombat</td>
											<td><img class="silhouette" src="images/clock/phylopic/Vombatus.svg" /></td>
										</tr>
										</tr>
										<tr>
											<td>Australidelphia (Australian)</td>
											<td>Diprotodontia</td>
											<td><i>Petaurus</i></td>
											<td>Sugar glider</td>
											<td><img class="silhouette" src="images/clock/phylopic/Petaurus.svg" /></td>
										</tr>										<tr>
											<td>Australidelphia (Australian)</td>
											<td>Diprotodontia</td>
											<td><i>Phascolarctos</i></td>
											<td>Koala</td>
											<td><img class="silhouette" src="images/clock/phylopic/Phascolarctos.svg" /></td>
										</tr>
										<tr>
											<td>Australidelphia (Australian)</td>
											<td>Diprotodontia</td>
											<td><i>Trichosurus</i></td>
											<td>Brushtail possom</td>
											<td><img class="silhouette" src="images/clock/phylopic/Trichosurus.svg" /></td>
										</tr>
										<!--tr>
											<td>Peramelemorphia</td>
											<td>Notoryctemorphia</td>
										</tr-->
										<tr>
											<td>Australidelphia (Australian)</td>
											<td>Dasyuromorphia</td>
											<td><i>Antechinus</i></td>
											<td>Antechinus</td>
											<td><img class="silhouette" src="images/clock/phylopic/Antechinus.svg" /></td>
										</tr>
										<tr>
											<td>Australidelphia (Australian)</td>
											<td>Dasyuromorphia</td>
											<td><i>Sarcophilus</i></td>
											<td>Tasmanian devil</td>
											<td><img class="silhouette" src="images/clock/phylopic/Sarcophilus.svg" /></td>
										</tr>
									</tbody>
									<tfoot>
										<tr>
											<td colspan="5" style="font-size: smaller; font-style: italic; text-align: right;">
												Silhouettes: <a href="https://www.phylopic.org/permalinks/a888ee7faeb10760a564b06efe167423ec9023129e1afa938434a53c7f3050e8" target="_blank">Phylopic contributors</a>
											</td>
										</tr>
									</tfoot>
								</table>
								<li>
									Clear the taxonomic name box and add expand the taxonomy tree
									to display the placental mammals.
									Select two taxa from each of the three major placental clades:
									<ul>
										<li>
											Afrotheria (African mammals)
										</li>
										<li>
											Laurasiatheria (
												Glires (rabbits & rodents)
											 + Euarchonta (tree shrews, flying lemurs + primates)
										 	)
										</li>
										<li>
											Euarchontoglires (
												insectivores, bats, ungulates;
												pangolins + carnivores
											)
										</li>
									</ul>
									Laurasiatheria and Euarchontoglires together form a clade.
									The "well-corroborated trees" in
									<a href="https://academic.oup.com/view-large/figure/362010550/syab072f1.tif"
									target="_blank">Asher &amp; Smith (2023)</a> give a
									detailed depiction of universally accepted relationships.
								</li>
								<li>
									Finally, add both available species of monotreme to your cart.
									The platypus and echidna will together form our outgroup.
								</li>
								<li>
									Open your cart (trolley icon, should read e.g. “12 items”).
									<ul><li class="caution">
										Note: Due to a bug with the website, if you close your cart window,
										 you may not be able to open it again without refreshing the page and
										 losing your progress.
									 </li></ul>
								</li>
								<!--li>
									Within the Cart window, Select “Protein alignment → one sequence per gene”.
								</li>
								<li>
									<p>
										Click “Advanced parameters” (under the "Align" button")
										to view the gap opening penalties.
									</p>
									<p>
										If a mismatched amino acid pair is penalized with one penalty
										point, this penalty could be avoided by adding a gap:
										<code>AAC</code> and <code>ACC</code> could be aligned
										with no penalty by inserting gaps to produce an aligned
										pair
										<code>AA-C</code> and <code>A-CC</code>, and making this
										gap really long could allow alignments to open long gaps
										instead of aligning stretches of a sequence that are mostly
										similar but differ in a number of places.
									</p>
								</li>
								<li>
									Click "Align" to start computing an alignment.
									This will take a few seconds as the algorithm tries to find
									the best fit match of AA sequences.
								</li>

								<li>
									Observe the graphical overview of the alignment.
									Note the highly conserved nature of the amino acid sequences.
									Scroll down to see the aligned sequences themselves.
								</li>
								<p>
									So far we've been aligning the amino acid sequences.
									We could work backwards from this alignment to produce a
									DNA alignment.  But we're instead going to use the mRNA
									nucleotide sequences directly.
								</p>
								<li>
									Close the COBALT alignment tool and return to your Cart.
								</li-->
								<li>
									Click "Download" → "RefSeq transcripts (FASTA)"
									(one sequence per gene). This will download the mRNA
									sequences that encode the catalase gene in these taxa.
								</li>
								<li>
									Open the downloaded file in a text editor (I recommend
									<a href="https://notepad-plus-plus.org/downloads/" title="Download Notepad++"
									target="_blank">Notepad++</a>;
									Apple users might prefer
									<a href="https://zed.dev/" target="_blank" title="Zed editor">Zed</a>)
								</li>
								<li>
									Perform a search & replace to replace spaces (<code> </code>)
									with underscores (<code>_</code>).
									<ul>
										<li>
											Optionally, remove other information, to leave only
											the species names – this doesn't affect the analysis,
											but may make your results easier to read.
										</li>
										<li>
											Be sure to retain the initial &ldquo;<code>&gt;</code>&rdquo;,
											such that
											<p><code>&gt;NM_001035386.2_Bos_taurus_catalase_(CAT),_mRNA</code></p>
											becomes <p><code>&gt;Bos_taurus</code></p>
										</li>
									</ul>
								</li>
							<h3>Align DNA sequences</h3>
							<p>Now we are ready to align our nucleotide sequences.</p>

								<li>
									Open the <a href="https://www.ebi.ac.uk/jdispatcher/msa/muscle"
									title="MUSCLE multiple sequence alignment" target="_blank">
									MUSCLE web tool</a>
								</li>
								<li>
									Under "Input sequence", select "Choose file" to upload your
									edited FASTA file.
									<ul>
										<li>
											Not working? Try my
											<a href="assets/clock/cat_refseq_transcript.fasta">sample file</a>.
										</li>
									</ul>
								</li>
								<li>
									Under "Parameters", select "Output Format: Pearson/FASTA".
								</li>
								<li>
									Click "Submit" and wait for your results to be ready.
								</li>
								<li class="extension">
									Whilst the alignment runs, consider the following questions:
								</li>
								<!--li class="question" onclick="Reveal('gap-cost');">
									Suggest a justification for the gap opening/extension values.
								</li>
								<li class="bulb">
									Consider what an alignment might look like if the "Opening" cost
									was reduced / increased.
									How would modifying the "Extension" cost change the alignment?
								</li>
								<li class="hidden written answer" id="gap-cost">
									<p>
										Cheap gap opening penalties allow slightly different sequences
										to be aligned with gaps rather than their homologous sequences,
										so we miss out on the evolutionary information we would
										obtain by comparing apples to apples.
									</p>
									<p>
										Make opening a gap too expensive, however, and real
										insertion/deletion events will not be reconstructed.
									</p>
									<p>
										Cheap extension penalties mean that once a gap is open,
										we may as well extend it indefinitely to swallow the
										other sequence, and avoid comparing bases that are
										different.
									</p>
									<p>
										Overall, we are trying to strike a balance between these
										factors, but once sequences start to diverge, and
										particularly as they accumulate insertion/deletion
										events, it's very unlikely that any alignment will be fully
										accurate.
									</p>
								</li-->
								<li class="question" onclick="Reveal('aln-correct');">
									Can we know whether an alignment is "correct"?
									(What would this mean?)
								</li>
                <li class="hidden written answer" id="aln-correct">
									The "correct" alignment matches the pattern of mutations that
									historically occurred, and is impossible to know for sure
									without travelling in time and watching evolution happen.
									The best we can do is to find an "optimal" alignment, according
									to whatever parameters we set.
								</li>
								<li class="question" onclick="Reveal('bad-aln');">
									If the alignment is "incorrect",
									what will the consequences be for inference?
								</li>
								<li class="hidden written answer" id="bad-aln">
									If two bases are different because they are misaligned,
									we will be inferring changes where none occurred.
									Our reconstruction of evolutionary relationships and rates
									will be misled. 
								</li>
								<li>
									Explore your alignment: switch to the "Alignments" tab, then zoom and drag using the axis.
								</li>
								<img src="images/clock/muscle-alignment.png" />
								<li class="extension">
									Notice that the MUSCLE algorithm aligns the sequences assuming
									a phylogenetic explanation for differences; it thus generates
									an <strong>unrooted</strong> phylogenetic tree as part of
									finding the best alignment. View this tree on the
									"Phylogenetic Tree" tab (scroll to "Phylogram").
								</li>
								<li class="question" onclick="Reveal('tree-root');">
									How would the tree look different if a root was specified?
								</li>
								<li class="hidden written answer" id="tree-root">
									The relationships would remain the same, but the outgroup
									would be at the bottom, making a different set of taxa
									appear monophyletic.  This doesn't affect relationships or
									sequence similarity; it does affect dates.
								</li>
								<li class="question" onclick="Reveal('rel-corr');">
									How do the relationships in your tree compare to those in
									<a href="https://academic.oup.com/view-large/figure/362010550/syab072f1.tif"
									target="_blank">well-corroborated trees</a>?
								</li>
								<li class="hidden written answer" id="rel-corr">
									If they aren't similar, we might start to worry about the
									quality of the alignment or the suitability of this gene
									for analysis.
								</li>
								<li>
									Download the results in FASTA format by heading to "Tool Output→Download".
									<ul>
										<li>
											If that doesn't work, try "Result Files→Alignment in FASTA format"
										</li>
										<li>
											If necessary, use my
											<a href="assets/clock/cat_aligned.fasta">sample output file</a>.
										</li>
									</ul>
								</li>
								<li>
									Rename the file so it has a <code>.fasta</code> extension
									(replacing the default <code>.txt</code>).
									<ul>
										<li class="bulb">
											Stuck?
											Try asking e.g.
											 <a href="https://claude.ai/chat/" target="_blank">Claude</a>,
											<a href="https://chat.openai.com/" target="_blank">ChatGPT</a>
											 "How do I change a file extension in Windows 11?"

											 Generative AI often gets to the point faster than
											 <acronym title="Search Engine Optimization">SEO</acronym>–bloated
											 <a href="https://www.indeed.com/career-advice/career-development/how-to-change-file-type" target="_blank" rel="nofollow">web search results</a>.
										 </li>
									 </ul>
								</li>
								<h3>Design a molecular clock analysis</h3>
								<li>
									If you haven't already,
									download BEAST2 and unzip the contents to a folder on your computer.
								</li>
								<li>
									Navigate to this folder and run "BEAUti".
									You may need to approve a security prompt.
								</li>
								<li>
									Import your alignment file using "File→Import alignment"
									<ul>
										<li class="bulb">
											If you see "Unsupported alignment",
										  check that you changed the file extension to <code>.fasta</code>
										</li>
								  </ul>
								</li>
								<img src="images/clock/beauti-import.png" />
								<li>Confirm that the data type is "nucleotide"</li>
								<li>
									<p>
										Open the "Site Model" tab and select a Substitution Model.
									</p>
									<p>
										The default, JC69, assumes a constant mutation rate between
										any of the four nucleotide bases.
									</p>
									<p>
										The HKY model incorporates the fact that chemically similar
										bases are easier to substitute – and tend to encode chemically
										similar amino acids, so mutations are less likely to be
										deleterious.
										The TN93 model fine-tunes this further by adding further
										parameters, whereas the fully generalized GTR
										(Generalized Time Reversible) model allows a distinct mutation
										rate between each different pair of nucleotides (64 in total) –
										whilst potentially more accurate, it takes more data to infer
										the values of all 64 parameters, making the model succeptible
										to overfitting with smaller datasets like ours.
									</p>
									<p>
									 	Select the HKY model and set Frequencies: Empirical.
								 	</p>
								</li>
								<li>
									Open the "Clock Model" tab.  We will use a Strict Clock
									for our initial analysis. Leave the Mean clock rate at its
									default setting (1).
								</li>

								<li>
									Open the "Priors" tab and update the "Tree.t" prior from
									"Yule Model" to "Calibrated Yule Model"
								</li>
								<li>
									Save your progress (File → Save)
								</li>
								<h4>Callibrate node ages with priors</h4>
								<li>
									At the bottom of the "Priors" tab, click "+ Add Prior".
								</li>
								<p>We are going to constrain the age of a clade.
									Our prior first dictates that a clade exists; then
									allows us to set a probability distribution on the
									date of its most recent common ancestor
									(<acronym title="most recent common ancestor">MRCA</acronym>).
								<li>
									If prompted for a prior type, select "MRCA Prior".
									<ul><li class="bulb"><strong>Be careful!</strong>
									Priors cannot be deleted – if you select the wrong type,
									you'll have to start again from your last save file.
								        </li>
									<li class="info">
										If your version of BEAST does not display this prompt, proceed to the next step.
									</li>
									</ul>
								</li>
								<li>
									Select <em>all</em> members of the group you wish to constrain.
									We'll start with the monotremes, <i>Ornithorhynchus</i> and
									<i>Tachyglossus</i>.  Move these taxa to the right-hand
									list (by pressing "&gt;&gt;").
								</li>
								<li>
									Enter "Monotremes" as a taxon set label
								</li>
								<li>
									Click "OK".
								</li>
								<p>
									The prior should now appear on your "Priors" tab – but as yet,
									it will have no effect on your analysis.
								</p>
								<li>
									Tick the "Monophyletic" checkbox.
								</li>
								<p>
									The prior now causes any trees in which these taxa do not
									form a clade to be discarded (i.e. assigned zero likelihood)
									during the analysis.
								</p>
								<li>
									Now that the "Monophyletic" checkbox has defined the taxa
									as a clade, we can constrain the date of their MRCA.
									Choose a prior from the the "[none]" drop-down box.
									A "Gamma" prior is probably appropriate here.
								</li>
								<li>
									Select the &#x25B6; button to expand the prior.
									Specify appropriate values for the "Alpha", "Beta" and
									"Offset" parameters.
								</li>
								<img src="images/clock/monotreme-prior.png" />
								<h4>Specifying a prior</h4>
								<p>
									This is a good point to ask what an appropriate prior
									distribution might look like.
									Bayesian priors attempt to encapuslate our prior beliefs,
									i.e. what we think we know before we run the analysis.
								</p>
								<p>
									At this point you probably don't have much prior
									knowledge about monotreme origination.  But such information
									is seldom far away; a Google Scholar search for
									“<a href="https://scholar.google.co.uk/scholar?q=Oldest+monotreme" target="_blank">
									Oldest monotreme</a>” throws up a 2008 PNAS article
									(fourth result for me)
									“<a href="https://dx.doi.org/10.1073/pnas.0706385105" title="The oldest platypus and its bearing on divergence timing
									of the platypus and echidna clades" target="_blanks">
									The oldest platypus and its bearing on divergence timing
									of the platypus and echidna clades</a>”.
								</p>
								<p>
									This study is perfect for our needs:
									it reports a total-group platypus, which is by definition a
									crown-group member of the clade we have constrained
									(strictly, Platypus + Echidna – there may be other crown-group
									monotremes not included in our analysis).
									This fossil is dated to 121–112.5&nbsp;Ma, so the MRCA of
									platypus and echidna <strong>must</strong> be older than 112.5&nbsp;Ma;
									this gives us a value for our "Offset" parameter.
								</p>
								<p>
									The other parameters are more subjective.
									This this is a rather derived fossil and
									(based on the PNAS abstract and the snippets in the other Google Scholar results)
									the fossil record of monotremes seems to be quite sparse,
									so I think it’s quite likely that there are older fossils
									remaining to be discovered – it would be surprising if the
									only relevant fossil we'd found was so close in time to the MRCA.
								</p>
								<p>
									The Beta parameter allows us to scale ("stretch") the distribution.
									The default value of 2 seems too small for me;
									our default priors assign a 95% probability to the MRCA
									being younger than 122&nbsp;Ma (see the Quantile report under
									the curve visualization), but our dating uncertainty on the
									fossil itself spreads back to 121&nbsp;Ma.
									Let's try a larger value: perhaps 200?
								</p>
								<p>
									Does this prior look reasonable?  Check the shape of the
									curve and its summary statistics.
									The very earliest stem-group mammal fossils are ~320&nbsp;Ma,
									so it's highly unlikely that the
									platupus-echidna MRCA dates back that far –
									The mammal crown group is probably not older than
									Morganucodonts, late Triassic (205&nbsp;Ma) upper-stem Mammals,
									so we probably don’t want a prior that assigns much probability
									to ages older than 205&nbsp;Ma for our monotreme divergence node.
									Perhaps a Beta value of 20 looks more reasonable?
								</p>
								<p>
									We should also consider that the fossil is quite derived,
									and that the ~30&nbsp;Ma gap to the second-oldest platypus
									fossil suggests that the monotreme fossil record is rather
									incomplete.
									These considerations can be reflected in our Alpha parameter.
									Compare values of 1 and 10 to get a feeling for how
									this parameter changes the shape of the probability
									distribution.
									If we felt it was likely that our constraining fossil was
									very close to the date of the MRCA, we might pick a small
									value of alpha; in this case, I am inclined to opt for a
									larger value of Alpha, which increases the mean of the
									distribution – simply put, it moves the "hump" of the
									distribution further back in time.
								</p>
								<p>
									Increasing Alpha also extends the tail of the distribution,
									so Beta will need adjusting accordingly: I'm happy with values
									of Alpha = 3; Beta = 10; Offset = 112.5.
								</p>
								<p>
									A vertebrate palaeontologist with a better understanding
									of platypus morphology and mammal taphonomy would no doubt
									come up with a different set of values.
									If we have lots of informative data, we hope that the signal
									will outweigh the prior: we update our beliefs in light
									of new evidence.
								  But if our data are not particularly informative, we might
									find that our results simply echo our priors back to us:
									in which case, justifying the precise shape of our prior
									distribution becomes even more important.
								</p>
								<li>
									Select at least one, ideally two or three, further clades to
									constrain.  Search the literature to justify your design
									of priors.  You may wish to complement the fossil record
									with palaeogeographical information: for instance,
									Austrailia separated from the Gondwanan continent around
									30&nbsp;Ma.
								</li>
								<li>
									To root the tree, add an MRCA prior including everything that's
									not a monotreme. Label it appropriately
									("Theria", or simply "ingroup").
									Leave the prior type to "[none]" and select "monophyletic".
									This will force the tree to be rooted on monotremes.
									Incidentally, it will also cause the analysis to report
									a posterior estimate of the date of the MRCA of therians
									(placentals and marsupuals).
								</li>
								<li>
									Finally, configure the
									<acronym title="Monte Carlo Markov Chain">MCMC</acronym>
									analysis.  Set the Chain Length to 100&nbsp;000.
									Simply put, this will try 100k different parameter variations
									(e.g. mutation rate, tree topology, branch lengths),
									sampling more plausible values of the parameters with higher
									frequencies.
								</li>
								<li>
									Safe your settings (File→Save).
								</li>
								<h3>Conduct the analysis</h3>
								<li>Return to your BEAST folder and launch BEAST.</li>
								<li>
									Select the "Input file" you just created in BEAUti.
									<ul><li class="bulb">
										Work from a <a href="assets/clock/cat_initial.xml">template</a>
										if necessary
									</li></ul>
								</li>
								<li>Notice the box reading "default: only write new log files".
									<ul><li class="bulb">
										Log files will be named based on the input file;
										if you wish to overwrite or add to a previous run, select
										the relevant option, otherwise the presence of previous log
										files will cause the run to abort with an error.
									</li></ul>
								</li>
								<li>Press "Run" to begin the analysis.</li>
								<li>
									Open Tracer.
									<ul><li class="bulb">
										This is not in the BEAST folder &ndash; you should have
										downloaded it separately; see the "Before you start"
										instructions at the top of this page.
									</li></ul>
								</li>
								<li>
									Under "Trace Files:", press "+" to load your BEAST log.
									<ul><li class="bulb">
										If the run's not finished,
										Tracer will display the current results; press the "Reload"
										button to update.
									</li>
									<li>
										Use a <a href="assets/clock/cat_initial.log">template file</a>
										if needed.
									</li></ul>
								</li>
								<li>
									Note the Estimated Sample Size for each parameter.
									This is an indication of how well the range of possible values
									has been explored; values &lt; 200 indicate that reconstructed
									values are unlikely to be statistically robust.
								</li>
								<li>
									Click on the "likelihood" Statistic in the Traces: list.
									Select the "Trace" tab and untick "Draw line plot"
									to view the likelihood of successive samples.
								</li>
								<img src="images/clock/tracer-likelihood.png" />
								<p>
									When a run has converged on the posterior distribution,
									the samples should plateau to form a cloud about a steady
									likelihood value. This shows that likely parameter values,
									are sampled often; and unlikely values
									(giving a lower likelihood of the model given the data)
									are sampled less often, in proportion to their probability.
								</p><p>
									The low values early in this run suggest that it took a
									while for the run to converge on a sensible set of parameters.
									These early results should be "burned off" and ignored,
									or they will skew the sample; the simplest solution
									is to run the chain for longer, which will also increase our
									sample size.
								</p>
								<li>
									Return to BEAUti and increase the MCMC Chain Length.
									2&nbsp;000&nbsp;000 should be enough.
								</li>
								<li>
									Save the BEAUti configuration. Use the same filename.
								</li>
								<li>
									Launch BEAST, select your file, and change from "default" to
									 "resume" to append the 2M new steps to your previous chain.
								 </li>
								 <li>
									 Whilst the analysis is running, return to Tracer and "Reload"
									 the results.
								 </li>
								 <p>
									 In the example results below, likelihood values have formed
									 a plateau, with the initial low values in the greyed-out
									 "burn-in" stage; and the <acronym title="Estimated Sample Size">
										 ESS</acronym> values are now mostly above 200.
										 Our results may now be meaningful.
								 </p>
								 <p>
									 If your run is not displaying this behaviour, or you are
									 seeing very low ESS values, check your model carefully for
									 errors or inconsistency in your priors; e.g. did you forget
									 to mark a clade as monophyletic, or specify unrealistically
									 young/old dates for a clade's MRCA?
								 </p>
								 <img src="images/clock/tracer-likelihood-2M.png" />
								 <h3>Interpret results</h3>
								 <li>
									 Switch to the "Estimates" tab and select the
									 mrca.age(Monotremes) Statistic from the Traces: list.
								 </li>
								 <p>
									 This displays the <strong>posterior</strong> probability of
									 the age of the MRCA of platypus and echidna.
									 Notice that this rather resembles our prior distribution:
									 were the data uninformative here?
								 </p>
								 <img src="images/clock/tracer-monotremes.png" />
								 <li>
									 Return to the BEAST folder and launch DensiTree.
								 </li>
								 <li>
									 Open the .trees file corresponding to your analysis.
									 The file name will be
									 <code>{BEAUti-filename}-{alignment-filename}.trees</code>;
									 see <a href="assets/clock/cat_initial-cat_aligned.trees">example</a>.
								 </li>
								 <p>
									 DensiTree overlays each sampled trees.  As trees are
									 sampled according to their probability, the more densely
									 shaded results are more likely.
								 </p>
								 <li>
									 Use Style:Triangle to view relationships. It may help to
									 also Show→Consensus Trees or Show→Root Canal.
								 </li>
								 <img src="images/clock/densitree-triangle.png" />
								 <li>
									 Use Style:Block and Grid: Full grid to view the date
									 estimates of key nodes.  This is an indirect way of viewing
									 the information available in Tracer – but Tracer can only
									 produce these histograms for constrained nodes.
								 </li>
								 <img src="images/clock/densitree-block.png" />
								 <li>
									 If the relationships recovered do not match those of the
									 well-constrained tree, you may need to add more constraints
									 by repeating the analysis after adding "monophyletic" MRCA
									 priors in BEAUti.  Try constraining Eutheria and Metatheria
									 to be monophyletic; use a different file name so you can
									 distinguish your results.
								 </li>
								 <img src="images/clock/densitree-strict.png" />
								 <li>
									 Compare the estimates of the divergence times to your priors.
									  Have your data been informative?
									</li>
									<li>
										Now look at the absolute values of your divergence time
										estimates, especially at the base of the ingroup.
										Do these look realistic?
									</li>
									<p>
										Perhaps our assumption of a "strict" clock, in which
										mutation rate is constant across every branch of the tree,
										was too strong an assumption.
										<a href="https://doi.org/10.1038/d41586-022-00976-w"
										title="Nature News & Views: Mutational clocks tick differently across species"
										target="_blank">
											Species-by-species differences
										</a> might cancel each other out, simply introducing noise;
										but if mammals had an early burst of radiation,
										or radiated rapidly after the K-Pg extinction, their rates
										may have been systematically higher for some of their history.
										This would make a strict clock predict an earlier origin.
								 </p>
								 <li>
									 Return to BEAUti and set the Clock Model to
									 "Optimized Relaxed Clock".
								 </li>
								 <li>
									 Save the settings with a different filename.
								 </li>
								 <li>
									 Run a BEAST analysis using this settings file.
								 </li>
								 <li>
									 Use Tracer to verify that your run has converged, and has
									 obtained a sufficient sample size (ESS > 100 for
									 relevant parameters).
									 As the relaxed clock has a larger parameter space (why?),
									 you may need to extend the run, perhaps up to 5M generations?
								 </li>
								 <li>
									 Compare the age of key clades to the priors you specified.
									 Are the data informative?
								 </li>
								 <h3>Evaluate influence of priors</h3>
								 <li>
									 Select the "Sample From Prior" box on BEAUti's MCMC
									 tab, and save the settings to a new file.
								 </li>
								 <li>
									 Run this analysis in BEAST. This runs the analysis without
									 any data, to explore the <strong>effective</strong> priors.
									 This captures any interactions between your initial prior
									 distributions.
								 </li>
								 <li>
									 Load the results into Tracer.  Compare the effective priors
									 with the posteriors you obtained when you ran the model using
									 the data.
								 </li>
								  <li class="question" onclick="Reveal('prior-comp');">
										How much have the data influenced your results?
									</li>
									<li class="hidden written answer" id="prior-comp">
										<p>
											If the effective prior is similar to the posterior, then
											our analysis is just spitting out what we put in;
											the data have not informed our inferrence.
										</p>
										<p>
											Be sure to compare the effective prior with the prior
											you originally specified.
											There's an interaction effect here: as marsupials are a
											subgroup of all mammals, marsupials can't be older
											than the MRCA of mammals.
											As such, a very long-tailed prior on the date of the
											marsupial MRCA might be curtailed by a narrow prior on the
											mammal MRCA.
										</p>
								 </li>
								 <li class="do">
									 The influence of priors on molecular clock results
									 is an important, but often overlooked, issue,
									 as highlighted by
									 <a href="https://dx.doi.org/10.1093/sysbio/syad057"
									 title="Systematic Biology: Two Notorious Nodes: A Critical Examination of Relaxed Molecular Clock Age Estimates of the Bilaterian Animals and Placental Mammals"
									 target="_blank">Budd and Mann (2023)</a>.
									 You should read the accessible
									 <a href="https://doi.org/10.1038/d41586-023-03487-4"
									  title="Nature News & Views: Genes are often uninformative for dating species’ origins"
										target="_blank">commentary</a> on this paper published
										in <i>Nature</i> News &amp; Views.
								 </li>
							</ol>
            </div>
          </div>
        </div>
      </div>
    </section>

    <!--section id="q-and-a" class="main">
      <header>
        <div class="container">
          <span class="feature-icon">
            <span class="icon solid fa-question"></span>
          </span>
          <h2>Questions</h2>
        </div>
      </header>
      <div class="content">
        <div class="container">
          <p>
            Propose and vote for questions or topics to cover during the
            face-to-face session.
          </p>
          <iframe src="https://app.sli.do/event/nQyN7PoeaWTDJp8QuAWwCW/questions"
                  height="100%" width="100%" style="min-height: 560px;">Loading…</iframe>
        </div>
      </div>
    </section-->

		<!-- Footer -->
    <section id="footer">
      <div class="copyright">
        <ul class="menu">
          <li>Content: <a href="http://smithlabdurham.github.io/">Martin R. Smith</a></li>
          <li>&copy; 2024 Durham University</li>
          <li>Design: <a href="http://html5up.net" rel='noopener'>HTML5 UP</a></li>
        </ul>
      </div>
    </section>

		<!-- Scripts -->
    <script src="assets/js/jquery.min.js"></script>
    <script src="assets/js/jquery.scrolly.min.js"></script>
    <script src="assets/js/browser.min.js"></script>
    <script src="assets/js/breakpoints.min.js"></script>
    <script src="assets/js/util.js"></script>
    <script defer src="assets/js/thisyear.js"></script>
    <script defer src="assets/js/main.js"></script>
	</body>
</html>
